www/docs/server/infer-types.md

```diff
@@ -15,81 +15,37 @@ It is often useful to wrap functionality of your `@trpc/client` or `@trpc/react`
 - `inferProcedureInput<TProcedure>`
 - `inferSubscriptionOutput<TRouter, TPath>`
```

Comment:
I think this should be removed

Reply:
Agreed!

Reply:
Ok. Wasn't sure about the API of the subscription so I just left it there for now. Can remove it. Anything else we should add?